Configuration/StandardSequences/python/Eras.py

```diff
@@ -70,6 +70,7 @@ def __init__(self):
                            'run3_HB', 'run3_HFSL', 'run3_common', 'run3_RPC',
                            'phase1Pixel', 'run3_GEM', 'run2_GEM_2017',
                            'run2_CSC_2018',
+                           'ecal_component','ecal_component_finely_sampled_waveforms',
```

Comment:
Since these Modifiers are not part of any Eras, maybe placing them in `Configuration/ProcessModifiers` would be more appropriate? (by the way, the python files defining the Modifers are missing)

Reply:
Thanks for suggestion @makortel 
I am working on the workflow. I will adapt the PR as you suggest.